### PR TITLE
Minor fix to user logging messages

### PR DESCRIPTION
**Pull request by Saphs**

### CODE REVIEW

**Review by manuelstolze — DISMISSED** [review]
LGTM!

File: src/controller/news.controller.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        destination: (req: Request, file: Express.Multer.File, cb) => {
+            cb(null, NewsController.storagePath);
+        },
+        filename: (req: Request, file: Express.Multer.File, cb) => {
+            cb(null, file.fieldname + '_' + Date.now() + path.extname(file.originalname));
+        }
+    });
+
+    private static multerLimits = {
+        fileSize: 1 * 1024 * 1024,
```
Comment (DanielLiebler) [inline src/controller/news.controller.ts:22]: why 1024^2?
why are limits set to 5 files, 10 parts?

File: src/controller/user.controller.ts [hunk 2]
```diff
@@ -210,8 +210,13 @@ class UserController {
             });
 
         } catch (error) {
-            console.error("Couldn't find a user with the following username: " + loginOptions.username + "\n" + "Error: ", error);
-            new APIResponse(res, 404, {}, ["Couldn't find a user with the provided username."]).send();
+            if (error.name === "EntityNotFound") {
+                console.log("(EntityNotFound) Couldn't find User: \"" + loginOptions.username + "\" with Password_hash: \"" + loginOptions.password + "\"");
+                new APIResponse(res, 404, {}, ["Username and Password did not match."]).send();
+            } else {
+                console.error("(EntityNotFound) Couldn't find a user with the following username: " + loginOptions.username + "\nError: ", error);
```
Comment (DanielLiebler) [inline src/controller/user.controller.ts:217]: if the error is NOT of type EntityNotFound, why log it as such..
I would like to see the error name be inserted here.

Comment (Saphs) [reply src/controller/user.controller.ts:217]: Ill change it

File: src/controller/news.controller.ts (hunk 1 above)
Comment (Saphs) [reply src/controller/news.controller.ts:22]: I will remove the whole multer setup as it is not nessecary if we will never allow user uploads in this context

Comment (manuelstolze) [reply src/controller/news.controller.ts:22]: LGTM!

File: src/controller/news.controller.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { Request, Response } from 'express';
+import { APIResponse } from "../../model/APIResponse";
+import { MIMETYPES } from "./mimeTypes";
+import fs from "fs";
+
+class NewsController {
+
+    private static storagePath = 'fileStash/news/'
```
Comment (manuelstolze) [inline src/controller/news.controller.ts:8]: Vlt in eine Env auslagern?

**Review by manuelstolze — APPROVED** [review]

File: src/controller/news.controller.ts (hunk 3 above)
Comment (Saphs) [reply src/controller/news.controller.ts]: done

**Review by manuelstolze — APPROVED** [review]
LGTM!

**Pull request merged**